app/coffee/HttpController.coffee

```diff
@@ -37,6 +37,23 @@ module.exports = HttpController =
 			size += (line.length + 1)
 		return size
 
+	getProjectDocs: (req, res, next = (error) ->) ->
+		project_id = req.params.project_id
+		# filter is string of existing docs "id:version,id:version,..."
+		filterItems = req.query?.filter?.split(',') or []
```

Comment:
Can we call this `ignore` or `exclude` instead. `filter` made me initially think it was the ones we wanted, no didn't want.